examples/01_plotting/plot_demo_more_plotting.py

```diff
@@ -135,24 +141,8 @@
 )
 
 # %%
-# Visualizing without a colorbar on the right side
-# ------------------------------------------------
-#
-# The argument ``colorbar`` should be set to ``False`` to show plots without
-# a colorbar on the right side.
-
-plotting.plot_stat_map(
-    stat_img,
-    display_mode="z",
-    cut_coords=1,
-    colorbar=False,
-    title="display_mode='z', cut_coords=1,\ncolorbar=False",
-    figure=plt.figure(figsize=(5, 7)),
-)
```

Comment:
did you move this somewhere?

Reply:
I was planning to put it in `plot_demo_plotting.py` however I realize that I finally ate it :/

Is it ok to put in `plot_demo_plotting.py`?

Reply:
yup